R/shared-methods.R

```diff
@@ -234,7 +244,12 @@ DataClass <- R6::R6Class(
       }
     },
 
-    #' @description Processes data.
+    #' @description Processes data by adding and calculating absent columns.
+    #' Called on clean data (after `clean`).
+    #' Some countries may have data as new events (e.g. number of
+    #' new cases for that day) whilst others have a running total up to that
+    #' date. Processing calculates these based on what the data comes with
+    #' via the functions `region_dispatch` and `process_internal`.
```

Comment:
can we list here the methods that are called by `process_internal` in a built point list (if you think this would be helpful) or perhaps in `process_internal`. It might be nice to mention that at some point we hope to make this more modular.